FunctorCategories/gap/Functors.gi

```diff
@@ -527,65 +527,69 @@ InstallMethod( EquivalenceFromFullSubcategoryOfProjectivesObjectsIntoAdditiveClo
     
 end );
 
-##
-BindGlobal( "SET_ISOMORPHISMS_BETWEEN_SOURCE_AND_IMAGE_OF_YONEDA_EMBEDDING_OF_SOURCE",
+if IsPackageMarkedForLoading( "ToolsForHigherHomologicalAlgebra", "" ) then
```

Comment:
I think strictly speaking one should keep the version check here. Otherwise one can theoretically load a very old version of `ToolsForHigherHomologicalAlgebra` which does not have the required functions manually before loading FunctorCategories and will get warnings.

Reply:
True

Reply:
I will wait until the ci-tests pass or fail and push again.